test/GasAccounting.t.sol

```diff
@@ -417,23 +471,23 @@ contract GasAccountingTest is Test {
         assertGt(bondedAfter, bondedBefore);
     }
 
-    function test_bundlerReimbursement() public {
-        initEscrowLock(gasMarker * tx.gasprice * 2);
+    // function test_bundlerReimbursement() public {
+    //     initEscrowLock(gasMarker * tx.gasprice * 2);
 
-        // The bundler is being reimbursed for the gas used between 2 gas markers,
-        // the first one is as the very beginning of metacall, and passed to _setAtlasLock(),
-        // the second one is in _settle().
+    //     // The bundler is being reimbursed for the gas used between 2 gas markers,
+    //     // the first one is as the very beginning of metacall, and passed to _setAtlasLock(),
+    //     // the second one is in _settle().
 
-        // First gas marker, saved as `rawClaims` in _setAtlasLock()
-        uint256 rawClaims = (gasMarker + mockGasAccounting.FIXED_GAS_OFFSET()) * tx.gasprice;
+    //     // First gas marker, saved as `rawClaims` in _setAtlasLock()
+    //     uint256 rawClaims = (gasMarker + mockGasAccounting.FIXED_GAS_OFFSET()) * tx.gasprice;
 
-        // Revert calculations to reach the second gas marker value in _settle()
-        (uint256 claimsPaidToBundler, uint256 netGasSurcharge) = mockGasAccounting.settle(solverOp.from, makeAddr("bundler"));
-        uint256 settleGasRemainder = initialClaims - (claimsPaidToBundler + netGasSurcharge);
-        settleGasRemainder = settleGasRemainder * mockGasAccounting.SURCHARGE_SCALE() / (mockGasAccounting.SURCHARGE_SCALE() + mockGasAccounting.SURCHARGE_RATE());
+    //     // Revert calculations to reach the second gas marker value in _settle()
+    //     (uint256 claimsPaidToBundler, uint256 netGasSurcharge) = mockGasAccounting.settle(solverOp.from, makeAddr("bundler"));
+    //     uint256 settleGasRemainder = initialClaims - (claimsPaidToBundler + netGasSurcharge);
+    //     settleGasRemainder = settleGasRemainder * mockGasAccounting.SURCHARGE_SCALE() / (mockGasAccounting.SURCHARGE_SCALE() + mockGasAccounting.SURCHARGE_RATE());
 
-        // The bundler must be repaid the gas cost between the 2 markers
-        uint256 diff = rawClaims - settleGasRemainder;
-        assertEq(diff, claimsPaidToBundler);
-    }
+    //     // The bundler must be repaid the gas cost between the 2 markers
+    //     uint256 diff = rawClaims - settleGasRemainder;
+    //     assertEq(diff, claimsPaidToBundler);
+    // }
 }
```

Comment:
need to open tickets to re-enable commented out tests

Reply:
Agreed